### ISSUE TITLE
Add sample questions

### ISSUE THREAD

**Issue by ben-ward1** [BenjaminEllisWard/RQG#3]
Sample questions can be added to Models/SampleQuestions.cs

**Comment by maxalmonte14** [comment]
I don't really understand this issue, there are already sample questions in the `SampleQuestions.cs` model, the idea is to add more? In that case, how many should be?

**Comment by ben-ward1** [comment]
Yes, I intended simply to add more. There can be as many as we can write, within reason.

The sample questions are used to populate the Db in the case that there are no existing questions. The implementation can be seen in QuestionsController.Index() (the default route). Based on this implementation, I do not see any limitations to how many can be added. Although, it is worth noting that entries are added asynchronously. If the sample questions array is sufficiently large and requires time to post to the Db, there may be some unexpected outcome if the user tries to use CRUD features while the async task is still executing, although I have not observed this behavior.

**Comment by maxalmonte14** [comment]
I personally think that any number between 5 and 10 is ideal, as these are sample questions it shouldn't be a lot of them, just enough to make the program operational.
